### DISCUSSION TITLE
Why can't I see extension reviews from other countries?

### DISCUSSION THREAD

**Discussion by Juraj-Masiar** [microsoft/MicrosoftEdge-Extensions#197] [General]
When I open any of my extension, for example this:
https://microsoftedge.microsoft.com/addons/detail/auto-highlight/dmpalknlndebnpmlmakoemcefnfbmkac
I see exactly 0 out of 7 reviews:

![image](https://github.com/user-attachments/assets/2c4cc7ab-931e-4af1-a7d3-5adc9f7bdbb5)

Is it because I'm from Slovakia? But we learn English from the first grade, so people here (and almost everywhere) speak English. And even if they don't they can use one of many translation addons to translate foreign reviews.

You should be able to see ALL reviews, not just those written by users from your country.
Either add a button/select box that will have option "All reviews" and "Only from your country".

To compare, Firefox Store shows all reviews, Chrome Store has "Language" select box with option "All languages" (also it uses your UI language as default, instead of your country of origin).

**Comment by ManikanthMSFT** [comment]
Hi @Juraj-Masiar, thank you for reporting this. We are looking into this and will let you know as soon as we have an update.

**Reply by ManikanthMSFT** [reply]
Hi @Juraj-Masiar, would like to inform you that this issue is on our priority list and will update you once we have more information.

**Reply by tarikrital** [reply]
Welcome, @ManikanthMSFT

Now one year passed is there a good news for this feature?

**Comment by Phero49** [comment]
am also having the same issue

**Reply by ManikanthMSFT** [reply]
Hi @Phero49, I would like to inform you that this issue is on our priority list and will update you once we have more information.

**Comment by venkatasai5171** [comment]
Hello,

please refer this https://github.com/microsoft/MicrosoftEdge-Extensions/issues/343
